ivy_lint/formatters/docs_formatter.py

```diff
@@ -0,0 +1,70 @@
+import re
+import ast
+
+from ivy_lint.formatters import BaseFormatter
+
+
+def format_docstring(doc):
+    """Formats a single docstring."""
+    # Rename "Functional Examples" to "Examples" and format it without the extra newline
+    doc = re.sub(r'(\s*)Functional Examples\n\1-*\n?', r'\1Examples\n\1--------\n', doc)
```

Comment:
Functional examples is only one issue, we may have others. I prefer if we limited the available sections instead. You may find common sections [here](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections). That way I think the code will be more generalized.

What do you think?

Reply:
@KareemMAX Yes, this approach is great since I also noticed some other incorrect names. I have now created a list of all valid sections and checking each section header if they belong to that list.